### ISSUE TITLE
[BUG] Title not centrally aligned when date and days are hidden

### ISSUE THREAD

**Issue by kalhimeo** [amaximus/garbage-collection-card#89]
**Describe the bug**
When using the card configuration to hide the date and days, the title is not aligned properly in the center of the card (vertically)

**To Reproduce**

```
type: custom:garbage-collection-card
entity: sensor.trash_normal
hide_date: true
hide_days: true
```

**Expected behavior**
The title should be centered vertically just like the icon

**Screenshots**
![image](https://user-images.githubusercontent.com/6355373/151658053-fccc29de-a7d0-44b1-8ca2-59882c522a2a.png)

**Environment (please complete the following information):**
 - Garbage Collection Card version: 2021.11
 - Browser: Edge for Windows, same behavior on tablets and probably with other browsers
 - Browser language: French and/or English

**Comment by amaximus** [comment]
@kalhimeo Thanks for reporting the issue.

Fixed in version 1.20.1 released just now.

Please update the card and get back to me with results.

**Comment by kalhimeo** [comment]
looks very good, thank you for the fast fix

![image](https://user-images.githubusercontent.com/6355373/151715458-54b1e061-2936-4969-b80e-ccbc73cbfd7b.png)
